appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/model/osv/request/QueryRequestPayload.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * -
+ * Copyright (C) 2023 Vaadin Ltd
```

Comment:
When I import the project in Eclipse for some reason it messes up the indentation of the header, while it does not if the comment block opens with `/*-`, e.g.

```suggestion
/*-
 * Copyright (C) 2023 Vaadin Ltd
```

Probably just an Eclipse thing, but let's use this syntax to avoid formatting issues for other users.